### ISSUE TITLE
Ability to configure it so it will ignore words inside js import paths

### ISSUE THREAD

**Issue by gnapse** [Jason-Rev/vscode-spell-checker#225]
Import file names are what they are, especially if they come from outside sources and packages not under the code author's control. Code will fail anyway if importing something from an invalid source path. So why bother checking that anyway? Is there a way to opt-in to ignore these lines, instead of having to add every single case to the dictionary?

```js
import { debounce } from '@ember/runloop';
import { assign } from '@ember/polyfills';
// "runloop" and "polyfills" are flagged, even though I have no control over that
```

I realize that my best bet would be to add them to either each Ember-based project's dictionary, or to my global dictionary. But isn't there a better way?

**Comment by Jason3S** [comment]
Thank you for the feedback.

There is a simple enough option for your case, add an ignoreRegExpList:

```javascript
// To exclude patterns, add them to "ignoreRegExpList"
"cSpell.ignoreRegExpList": [
	"/from\\s+(['\"]).*\\1/"
]
```

I am working on a general way of solving this problem. It is not an easy one.

The first approach I have to is make the spell checker grammar aware, similar to how the colorizer works. This will allow for the checking of variable names, function names, strings, and comments; while ignoring the rest.

The other approach is to allow words to be dynamically ignored based upon some rules.

**Comment by gnapse** [comment]
The regex setting is a good enough solution for me, thanks!
